### DISCUSSION TITLE
Order change by customer not possible

### DISCUSSION THREAD

**Discussion by mvanderkruk** [pretix/pretix#3833] [Q&A]
I have an issue with my pretix installation. I have enabled the option for users to be able to change their orders in the cancellation settings. For paid orders the option to only request a cancellation is enabled.
For some of the products the option "allow product to be cancelled or changed" is enabled.
On the order overview page, there is no option to change the order or request cancellation of the order.

When I enable the option "Customers can cancel their paid orders" There is only the option to cancel the complete order.

I want the customer to be able to change the order and request order cancellations. What am I missing?

**Comment by raphaelm** [comment]
Cancellation of individual products as a customer is not implemented, if that is what you are asking

**Reply by mvanderkruk** [reply]
What I would to accomplish is for a customer to add products to an order.
Both regular products and add-on products.

**Reply by raphaelm** [reply]
Adding regular products is also not implemented. Add-ons should work with correct configuration. Note that changes/cancellation must be allowed on *all* products in the order.
